File: backend/entity/Manager.go
package entity

import (
	"time"

	"gorm.io/gorm"
)

// delete
type Manager struct {
	gorm.Model

	Username  string
	Email     string
	Password  string
	FirstName string
	LastName  string
	Birthday  time.Time

	SaleList []SaleList `gorm:"foreignKey:ManagerID"`
}

File: frontend/src/interface/Manager.ts
export interface Manager {
  ID: number;
  CreatedAt: string;
  UpdatedAt: string;
  DeletedAt?: string | null;

  Username: string;
  Email: string;
  Password: string;
  FirstName: string;
  LastName: string;
  Birthday: string;

  SaleList?: unknown | null;
}
